components/src/dynamo/global_router/handler.py

```
@@ -23,6 +23,45 @@
+    for idx in pool_chain:
+        namespace = namespaces[idx]
+        client = clients[namespace]
+        try:
+            stream = await client.generate(request)
+            if last_err is not None:
+                logger.info(
+                    f"Routing {pool_kind} request: chain {pool_chain} -> "
+                    f"accepted by pool {idx} ({namespace}) after fallback"
+                )
+            return stream, idx
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)